template/.github/workflows/build.yml.j2

```diff
@@ -296,8 +296,10 @@ jobs:
     env:
       NEXUS_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
       HELM_REPO: ${{ needs.select_helm_repo.outputs.helm_repository }}
-      OCI_REGISTRY_PASSWORD: ${{ secrets.HARBOR_ROBOT_STACKABLE_GITHUB_ACTION_BUILD_SECRET }}
-      OCI_REGISTRY_USERNAME: "robot$stackable+github-action-build"
+      OCI_REGISTRY_STACKABLE_PASSWORD: ${{ secrets.HARBOR_ROBOT_STACKABLE_GITHUB_ACTION_BUILD_SECRET }}
+      OCI_REGISTRY_STACKABLE_USERNAME: "robot$stackable+github-action-build"
+      OCI_REGISTRY_STACKABLE_CHARTS_PASSWORD: ${{ secrets.HARBOR_ROBOT_STACKABLE_CHARTS_GITHUB_ACTION_BUILD_SECRET }}
+      OCI_REGISTRY_STACKABLE_CHARTS_USERNAME: "robot$stackable-charts+github-action-build"
```

Comment:
Will this change to `SDP` or should this PR be merged independent of the naming decision in https://github.com/stackabletech/infrastructure/issues/138?